ui/selectmenu.js

```diff
@@ -115,10 +115,13 @@ return $.widget( "ui.selectmenu", {
 			.insertAfter( this.element );
 
 		this._addClass( this.button, "ui-selectmenu-button ui-selectmenu-button-closed",
-			"ui-widget ui-state-default" );
+			"ui-button ui-widget" );
 
 		icon = $( "<span>" ).prependTo( this.button );
+		space = $( "<span> </span>" );
+		this._addClass( space, "ui-selectmenu-button-space" );
```

Comment:
This should probably be "ui-selectmenu-icon-space" its the space we insert before/after icons to properly align them.